l10n_fr_fec_oca/wizard/account_fr_fec_oca.py

```diff
@@ -0,0 +1,628 @@
+# Copyright 2013-2020 Akretion France (http://www.akretion.com)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# Copyright 2016-2020 Odoo SA (https://www.odoo.com/fr_FR/)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+# This module is a fork of l10n_fr_fec from official addons
+# (which itself was copied from OCA with my authorisation)
+# The construction of SQL requests don't respect pylint E8103
+# The problem is that fixing this would require large changes in the code
+# which would make this module a deeper fork of l10n_fr_fec
+# and would make it more difficult to compare the 2 modules and port
+# changes/improvements between each other
+# pylint: skip-file
+
+import base64
+import io
+import logging
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import float_is_zero
+
+logger = logging.getLogger(__name__)
+
+try:
+    from unidecode import unidecode
+except ImportError:
+    logger.debug("Cannot import unidecode")
+try:
+    import unicodecsv
+except ImportError:
+    logger.debug("Cannot import unicodecsv")
+
+
+class AccountFrFecOca(models.TransientModel):
+    _name = "account.fr.fec.oca"
+    _description = "Ficher Echange Informatise"
+
+    date_range_id = fields.Many2one("date.range", string="Date Range")
+    date_from = fields.Date(string="Start Date", required=True)
+    date_to = fields.Date(string="End Date", required=True)
+    encoding = fields.Selection(
+        [
+            ("iso8859_15", "ISO-8859-15"),
+            ("utf-8", "UTF-8"),
+            # ('cp500', 'EBCDIC'),
+            ("ascii", "ASCII"),
+        ],
+        string="Encoding",
+        default="iso8859_15",
+        required=True,
+    )
+    delimiter = fields.Selection(
+        [
+            ("|", "|"),
+            ("tab", "Tab"),
+        ],
+        default="|",
+        string="Field Delimiter",
+        required=True,
+    )
+    partner_option = fields.Selection(
+        [
+            ("types", "Selected Account Types"),
+            ("accounts", "Selected Accounts"),
+            ("all", "All"),
+        ],
+        default="types",
+        required=True,
+        string="Partner Export Option",
+    )
+    partner_account_type_ids = fields.Many2many(
+        "account.account.type",
```

Comment:
This model does not exist anymore.

Reply:
I can't understand how the CI can be green with such code line. Any idea ?

Reply:
@juliette-blanc Can you check the comments above ? (:

Reply:
@juliette-blanc Thanks for this PR. Could you check the comment above ?